components/avatar/avatar.css

```diff
@@ -0,0 +1,62 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
+
+body{
+    background-image: none;
+}
+.description{
+    margin-top: 1rem;
+    font-size: 1.5rem;
+}
+.avatar-main-container{
+    margin-left: 6rem;
+}
+.avatar-icon{
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+    padding-top: 2rem ;
+    display: flex;
+    justify-content: space-around;
+    height: 12rem;
+    width: 80%;
+    border-radius: 2px solid var(--alert-secondary-color);
```

Comment:
please check the syntax here, it should be border: 2px solid var(--alert-secondary-color);

Reply:
Fixed it. PLease review